hub.Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM gradle:jdk8 as builder
+
+COPY . /home/gradle/src
+USER root
+RUN chown -R gradle:gradle /home/gradle/src
+USER gradle
+
+WORKDIR /home/gradle/src
+RUN gradle assemble
+
+FROM openjdk:8-jre-alpine
+
+COPY --from=builder /home/gradle/src/build/libs/tribunals-case-api.jar /opt/app/
+
+WORKDIR /opt/app
+
+HEALTHCHECK NONE
+
+EXPOSE 8082
```

Comment:
sscs case loader uses port 8082. So it will be better to use TCA API default port of 8080

Reply:
Port updated to use 8080